### ISSUE TITLE
unable to use check method with class option

### ISSUE THREAD

**Issue by mattabedi** [teamcapybara/capybara#2027]
I'm upgrading a rails app from Capybara 2.18.0 to Capybara 3.0.2 and seem to be unable to use the `check` method with the `class:` option.

## Meta
Capybara Version: 3.0.2
Driver Information (and browser if relevant): selenium-webdriver 3.11.0 with chromedriver 

## Expected Behavior
Checkbox is selected

## Actual Behavior
```
Capybara::ElementNotFound: Unable to find visible checkbox {:class=>"test-checkbox-class"} that is not disabled
```

## Steps to reproduce

```
require "capybara"
 
html = DATA.read
app = proc { |env| [200, { "Content-Type" => "text/html" }, [html] ] }

sess = Capybara::Session.new(:selenium_chrome, app)
sess.visit("/")
sess.check(class: "test-checkbox-class")

__END__

<!DOCTYPE html>
  <html>
  <head>
  <title>CapybaraTestApp</title>
    
    
   i
  <link rel="stylesheet" media="all" href="/assets/application-35729bfbaf9967f119234595ed222f7ab14859f304ab0acc5451afb387f637fa.css" data-turbolinks-track="reload" />
  <script src="/assets/application-ca00fb0fac71019549fc5c2578a6a1e4127045442e9270ebebcc33130cb42568.js" data-turbolinks-track="reload"></script>
  </head>
    
  <body>
  <h1>New User</h1>
    
  <form action="/users" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="&#x2713;" />
    
  <div class="field">
  <label for="user_name">Name</label>
  <input type="text" name="user[name]" id="user_name" />
  </div>
    
  <div class="field">
  <label for="user_vip">Vip</label>
  <input name="user[vip]" type="hidden" value="0" /><input class="test-checkbox-class" type="checkbox" value="1" name="user[vip]" id="user_vip" />
  </div>
   
  <div class="actions">
  <input type="submit" name="commit" value="Create User" data-disable-with="Create User" />
  </div>
  </form>
    
  <a href="/users">Back</a>
    
  </body>
  </html>
```

**Comment by twalpole** [comment]
There's a bug in 3.0.x where the locator cannot be skipped with `check` that will be fixed in 3.0.3 or 3.1.0 whichever releases next - It's fixed in the master and 3.0_stable branches currently - https://github.com/teamcapybara/capybara/commit/e45ed4b8cd8a612d0c2fed43e9ddffb8d9869582.
As a workaround you can call `check(nil, class: "test-checkbox-class")`

**Comment by mattabedi** [comment]
Thanks @twalpole!
